riptide-core/src/main/java/org/zalando/riptide/NoopRoute.java

```diff
@@ -0,0 +1,33 @@
+package org.zalando.riptide;
+
+import org.apiguardian.api.API;
+import org.springframework.http.client.ClientHttpResponse;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+@API(status = STABLE)
+public final class NoopRoute implements Route {
+
+    private static final Route NOOP = new NoopRoute();
+
+    private NoopRoute() {
+
+    }
+
+    @Override
+    public void execute(
+            final ClientHttpResponse response, final MessageReader reader) {
+        // nothing to do
+    }
+
+    /**
+     * Produces a {@link Route} that ignores the response without actively
+     * discarding the response body.
```

Comment:
A hint like "It's the client's responsibility to close the `ClientHttpResponse`" may make sense explicit?